### ISSUE TITLE
Logon Authentication

### ISSUE THREAD

**Issue by akahn16** [dstaulcu/SystemEventsToSpeech#6]
Logon Authentication Successes & Failures can be spoken as JAWS doesn't do this.

**Comment by dstaulcu** [comment]
![image](https://cloud.githubusercontent.com/assets/6827994/20467507/2ccf3530-af57-11e6-83ef-d0a90f0d9905.png)

**Comment by dstaulcu** [comment]
Added (but commented out) sections in 1.3.0 to subscribe to and partially handle logon related events.

**Comment by dstaulcu** [comment]
after some testing it appears that there is significant delay in delivery/receipt of event subscriptions near to machine startup and user logon.  Might need to either defer this to windows 10 or evaluate non-event subscription based methods
